install.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+set -e
```

Comment:
We should be more strict. I'd suggest `set -e -u -o pipefail`

See https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425